cpp/bench/ann/src/cuvs/cuvs_mg_cagra_wrapper.h

```diff
@@ -18,7 +18,7 @@
 #include "cuvs_ann_bench_utils.h"
```

Comment:
these files in bench will go away after merged of [this PR](https://github.com/rapidsai/cuvs/pull/869)